src/components/Tooltip/internal/Tooltip.module.scss

```
@@ -0,0 +1,34 @@
+@use 'sass:map';
+@use '../../../style/variables/colors.scss';
+@use '../../../style/variables/spaces.scss';
+@use '../../../style/mixins/font-config.scss';
+
+.tooltip {
+  color: map.get(colors.$colors, 'base-white');
+  position: relative;
+  &:hover span {
+    visibility: visible;
+  }
+  span {
+    visibility: hidden;
+    background-color: map.get(colors.$colors, 'base-white');
+    color: map.get(colors.$colors, 'grey-dark');
+    position: absolute;
+    bottom: 100%;
+    border-radius: map.get(spaces.$spaces, '2');
+    margin: map.get(spaces.$spaces, '2');
+    padding: map.get(spaces.$spaces, '2');
+    margin-left: -1px;
```

Doing that make the end of the tooltip tail align to the icon ~~for some reason~~.
```suggestion
    margin-left: 4px;
```

---

Thank you :)